test/bridge/testpsi4.py

```diff
@@ -25,7 +25,7 @@ def test_makepsi4(self):
         a = np.array([[-1, 1, 0], [0, 0, 0], [1, 1, 0]], "f")
```

Comment:
Since we are already modifying the test can we change `a` to `atompos` or `atomcoords`